tests/tier1/tc_1045_check_rhsm_proxy_function_in_rhsm_conf.py

```diff
@@ -18,6 +19,12 @@ def test_run(self):
         config_name = "virtwho-config"
         config_file = "/etc/virt-who.d/{0}.conf".format(config_name)
         self.vw_etc_d_mode_create(config_name, config_file)
+        if "libvirt-local" in hypervisor_type:
+            register_config = self.get_register_config()
```

Comment:
duplicated line...

Reply:
done to remove duplicated and useless lines.